lib/rules/eol-last.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
                 const sourceCode = context.getSourceCode(),
                     src = sourceCode.getText(),
                     location = {
-                        column: 1,
+                        column: lodash.last(sourceCode.lines).length,
```

Comment:
@platinumazure Of course! Here you go:

### Unwanted EOL (CRLF)

Consider the following file:
```js
/* eslint eol-last: [1, "never"] */\r\n
var a = 2;\r\n
```
Again, those are three lines, and their lengths are (maybe surprisingly), again, `[35, 10, 0]`. By the same argument, again, column `0` is used. Thus, we get the exact same error report for the first example:

`  3:1  warning  Newline not allowed at end of file        eol-last`

### Missing EOL (CRLF)

Just like in the LF example, we have only two lines in this case:
```js
/* eslint eol-last: [1, "always"] */\r\n
var a = 2;
```

Again, `eol-last` uses column `10` and thus reports:

`  2:11  warning  Newline required at end of file but not found  eol-last`